src/sentry/ratelimits/base.py

```diff
@@ -2,9 +2,15 @@
 
 
 class RateLimiter(Service):
-    __all__ = ("is_limited", "validate")
+    __all__ = ("is_limited", "validate", "current_value", "is_limited_with_value")
 
     window = 60
 
     def is_limited(self, key, limit, project=None, window=None):
         return False
+
+    def current_value(self, key, project=None, window=None):
+        return 0
+
+    def is_limited_with_value(self, key, limit, project=None, window=None):
```

Comment:
@AniketDas-Tekky Could we add typing or doc-strings to these functions so the return values are more clear? I think `is_limited` makes sense, and `current_value` makes a bit of sense, but hearing `is_limited_with_value` doesn't immediately make me think it's going to return both, plus the expected order isn't obvious unless you jump over to this code.